src/SAWScript/Crucible/LLVM/X86.hs

```diff
@@ -502,12 +517,12 @@ allocateStack szInt = do
 
 -- | Process a crucible_alloc statement, allocating the requested memory and
 -- associating a pointer to that memory with the appropriate index.
-executeAllocation ::
```

Comment:
don't rename, or rename as part of a separate commit. I'm not requesting you to make this change in this pull request, but I will in future pull requests :-)